electricity/src/lib.rs

```diff
@@ -1,53 +1,150 @@
 extern crate reqwest;
 
 use reqwest::Error;
+use tokio_cron_scheduler::{JobScheduler, Job};
+use scraper::{Html, Selector};
+use db::{DATABASE, create_table};
+use uuid::Uuid;
+use aws_sdk_dynamodb::{
+    types::{AttributeValue},
+    Client, Error as DynamoDBError
+};
+
+pub mod db;
 
 struct ShutDownParseData {
-    days_number: i8,
     page_url: String,
     city: String,
 }
 
-pub async fn console_lib() -> Result<(), Error> {
+struct TableRowData {
+    city: String,
+    region: String,
+    time: String,
+    street: String,
+    date: String,
+}
+
+async fn fetch_page(url: &str) -> Result<String, Error> {
+    let response = reqwest::get(url).await?;
+    let body = response.text().await?;
+
+    Ok(body)
+}
+
+pub async fn collect_data() -> () {
     let dates = vec![
```

Comment:
Let's turn it into constant array and pass it as a dependency `dates: &[ShutdownParseData]` to the function.